[CI] PushQueriesIT testEqualityAndOther {SEMANTIC_TEXT_WITH_KEYWORD} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #10787 / rocky-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/sdeaudpigrqrw)
- [elasticsearch-pull-request #94421 / part-3](https://gradle-enterprise.elastic.co/s/ootxqinaujg2a)
- [elasticsearch-pull-request #94281 / part-3](https://gradle-enterprise.elastic.co/s/ecio34ygqsrvc)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.PushQueriesIT.testEqualityAndOther {SEMANTIC_TEXT_WITH_KEYWORD}" -Dtests.seed=3C4F3BA69F0490A6 -Dtests.locale=ky -Dtests.timezone=Etc/GMT+10 -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.PushQueriesIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testEqualityAndOther%20%7BSEMANTIC_TEXT_WITH_KEYWORD%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Expected a map containing
             operator: "LuceneSourceOperator[shards = [test:0], maxPageSize = 255, remainingDocs = 2147483646]"
               status: a map containing
      processed_queries: a list containing
                        0: expected ("#FieldExistsQuery [field=foo] #single_value_match(foo)" or "foo:[1 TO 1]") but was "*:*"
       processed_slices: <1> unexpected but ok
       processed_shards: <[test:0]> unexpected but ok
           total_slices: <1> unexpected but ok
              slice_max: <0> unexpected but ok
                current: <2147483647> unexpected but ok
              slice_min: <0> unexpected but ok
           rows_emitted: <1> unexpected but ok
          pages_emitted: <1> unexpected but ok
          process_nanos: <118474> unexpected but ok
partitioning_strategies: <{test:0=SHARD}> unexpected but ok
            slice_index: <0> unexpected but ok
```

**Issue Reasons:**
- [main] 3 failures in test testEqualityAndOther {SEMANTIC_TEXT_WITH_KEYWORD} (1.2% fail rate in 258 executions)
- [main] 2 failures in step part-3 (1.5% fail rate in 134 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (1.5% fail rate in 133 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -405,9 +405,6 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=analytics/nested_top_metrics_sort/terms order by top metrics numeric not null integer values}
   issue: https://github.com/elastic/elasticsearch/issues/135162
-- class: org.elasticsearch.xpack.esql.qa.single_node.PushQueriesIT
-  method: testEqualityAndOther {SEMANTIC_TEXT_WITH_KEYWORD}
-  issue: https://github.com/elastic/elasticsearch/issues/135333
 - class: org.elasticsearch.search.sort.FieldSortIT
   method: testSortMixedFieldTypes
   issue: https://github.com/elastic/elasticsearch/issues/129445

--- a/x-pack/plugin/esql/qa/server/single-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/PushQueriesIT.java
+++ b/x-pack/plugin/esql/qa/server/single-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/PushQueriesIT.java
@@ -181,7 +181,7 @@ public class PushQueriesIT extends ESRestTestCase {
                  * single_value_match is here because there are extra documents hiding in the index
                  * that don't have the `foo` field.
                  */
-                List.of("#FieldExistsQuery [field=foo] #single_value_match(foo)", "foo:[1 TO 1]");
+                List.of("FieldExistsQuery [field=foo]", "foo:[1 TO 1]");
         };
         ComputeSignature dataNodeSignature = switch (type) {
             case AUTO, CONSTANT_KEYWORD, KEYWORD, TEXT_WITH_KEYWORD -> ComputeSignature.FILTER_IN_QUERY;

